README.md

```diff
@@ -48,6 +48,8 @@ In package.json, escape quotes:
 
 Good frontend one-liner example [here](https://github.com/kimmobrunfeldt/dont-copy-paste-this-frontend-template/blob/5cd2bde719654941bdfc0a42c6f1b8e69ae79980/package.json#L9).
 
+With no parameters, concurrently will try to load it's configuration from **concurrently.json**.
```

Comment:
By applying the change suggested in https://github.com/kimmobrunfeldt/concurrently/pull/122/files#r144599446 we can remove this one (or at least fix it - `it's` => `its`!)